client/src/js/components/Chat.jsx

```diff
@@ -0,0 +1,157 @@
+import React, {Component} from 'react';
+
+// import { group } from '../authentication/authentication'
+// import { groupRef } from '../firebase/firebase';
+
+// Without Firebase
+class Group extends Component {
+
+  constructor(props, context) {
+    super(props, context)
+    this.updateMessage = this.updateMessage.bind(this)
+    this.submitMessage = this.submitMessage.bind(this)
+    this.state = {
+        message: '',
+        messages: [
+            {id:0, text: 'first message'},
+            {id:1, text: 'first message'},
+            {id:2, text: 'first message'}
+
+        ]
+    }
+    
+  }
+
+  updateMessage(event){
+    console.log('updateMessage:'+ event.target.value)
+    this.setState({
+      message: event.target.value 
+    })
+  }
+
+  submitMessage(event){
+    console.log('submitMessage: '+ this.state.message)
+    const nextMessage = {
+      id: this.state.messages.length,
+      text: this.state.message
+    }
+
+    var list = Object.assign([], this.state.messages)
```

Comment:
CRITICAL: Use `const` not `var`.

I'm not sure why you are using `Object.assign` here.